hw_3/t2.py

```diff
@@ -0,0 +1,15 @@
+import os
+os.system('cls')
+
+import random
+n = int(input("Введите количество сгенерированных чисел: "))
+arr = [random.randint(0, 6) for _ in range(n)]
+print(arr)
+X = int(input("Введите проверяемое число: "))
+min_diff = abs(arr[0] - X)
+closest_num = arr[0]
+for i in range(1, n):
+    if abs(arr[i] - X) < min_diff:
+        min_diff = abs(arr[i] - X)
```

Comment:
Чтобы не писать дважды `abs(arr[i] - X)` можно использовать моржовый оператор в условии
```suggestion
    if min_diff > (min_diff := abs(arr[i] - X)):
```

Reply:
Благодарю!